testing/snapshot-test.md

```diff
@@ -0,0 +1,20 @@
+# Snapshot test
+
+## Introduction
+Snapshot tests are used to test visual regression of presentational components and for reducers in order to avoid deep comparison boilerplate code
+
+## Dos
+- The visual component output has to be relatively small.  
+- Focus to shallow snapshot comparison.
+- In cases where snapshot output is not readable use enzyme-to-json utility.
+- Create snapshot for every presentational component.  
+
+## Don'ts
+- Don't use snapshots for testing component prop changes. Do assertions on specific DOM nodes directly.
+- Don't use mounted snapshots, better divide components to smaller ones and do shallow snapshots on them directly.
+
+## Process
+1) Create a presentational component  
+2) Create a snapshot with default props
+3) Check if snapshot is meaningful else use enzyme-to-json util
+4) If there is a need for different props test - assert on DOM nodes  
```

Comment:
A few suggestions:
1. You could include some examples of good tests + snapshots from the repo.
2. You could include something about reviewing snapshots. Snapshots should be reviewed with the same seriousness as reviewing code, and that's why it is important to keep them concise.
3. I find snapshots quite useful when writing unit tests too, as they reduce duplication. Maybe include something about that? Snapshot testing is not _only_ limited to React components

Reply:
Have included example in introduction. And added other stuff to notes